This patch of mine has been accepted upstream already.

commit a1cc71207cd22d7d884f44d135d975d981fa21e6
Author: Gerald Pfeifer <gerald@pfeifer.com>
Date:   Tue Aug 24 14:53:47 2021 +0200

    nsiproxy: Include sys/queue.h from tcp.c.
    
    In what looks like a change between FreeBSD 11 and later versions
    (FreeBSD 12 and 13), we now need to explicitly include sys/queue.h
    to avoid
    
      In file included from dlls/nsiproxy.sys/tcp.c:42:
      /usr/include/netinet/tcp_var.h:516:2: error:
      expected specifier-qualifier-list before 'TAILQ_ENTRY'
      516 | TAILQ_ENTRY(tcptw)  tw_2msl;
    
    Signed-off-by: Gerald Pfeifer <gerald@pfeifer.com>
    Signed-off-by: Huw Davies <huw@codeweavers.com>
    Signed-off-by: Alexandre Julliard <julliard@winehq.org>

--- UTC
diff --git dlls/nsiproxy.sys/tcp.c dlls/nsiproxy.sys/tcp.c
index 8567330be99..603aefadfe9 100644
--- dlls/nsiproxy.sys/tcp.c
+++ dlls/nsiproxy.sys/tcp.c
@@ -58,6 +58,10 @@
 #include <netinet/ip_var.h>
 #endif
 
+#ifdef HAVE_SYS_QUEUE_H
+#include <sys/queue.h>
+#endif
+
 #ifdef HAVE_NETINET_TCP_VAR_H
 #include <netinet/tcp_var.h>
 #endif
